sql/src/test/java/io/crate/operation/collect/LuceneDocCollectorTest.java

```diff
@@ -412,4 +456,142 @@ public void testOrderByFieldVisitorExpressions() throws Exception {
                 "{\"continent\":\"America\",\"countryName\":\"USA\",\"population\":1000}| 1000\n" +
                 "{\"continent\":\"America\",\"countryName\":\"USA\",\"population\":1001}| 1001\n"));
     }
+
+    private static void addDocToLucene(IndexWriter w, Long value) throws IOException {
+        Document doc = new Document();
+        if (value != null) {
+            doc.add(new LongField("value", value, Field.Store.NO));
+        } else {
+            // Create a placeholder field
+            doc.add(new StringField("null_value", "null", Field.Store.NO));
+        }
+        w.addDocument(doc);
+    }
+
+    private static Directory createLuceneIndex() throws IOException {
+        File tmpDir = Files.createTempDir();
```

Comment:
maybe use TempraryFolder rule here, to guarantee that it is deleted after the test has run